Dockerfile_build_ubuntu64_fluxtest

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
```

Comment:
Everything but the entrypoint here is a copy-paste from `Dockerfile_build_ubuntu64`

Reply:
Can we use the same Dockerfile and change the command we run instead? That feels nicer than lots of docker files with different entrypoints.